app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+
+      def index
+        @topics = Topic.all 
+        render json: @topics
+      end         
+    end
+  end
+end
```

Comment:
Add a line here.

Reply:
Do you mean at the end?

Reply:
Yes

Reply:
Actually there is an extra line at the end of the file